mysql/privs.go

```diff
@@ -56,41 +56,43 @@ var Priv2Str = map[PrivilegeType]string{
 
 // Priv2SetStr is the map for privilege to string.
 var Priv2SetStr = map[PrivilegeType]string{
-	CreatePriv:     "Create",
-	SelectPriv:     "Select",
-	InsertPriv:     "Insert",
-	UpdatePriv:     "Update",
-	DeletePriv:     "Delete",
-	DropPriv:       "Drop",
-	GrantPriv:      "Grant",
-	ReferencesPriv: "References",
-	LockTablesPriv: "Lock Tables",
-	AlterPriv:      "Alter",
-	ExecutePriv:    "Execute",
-	IndexPriv:      "Index",
-	CreateViewPriv: "Create View",
-	ShowViewPriv:   "Show View",
-	CreateRolePriv: "Create Role",
-	DropRolePriv:   "Drop Role",
-	ShutdownPriv:   "Shutdown Role",
+	CreatePriv:         "Create",
+	SelectPriv:         "Select",
+	InsertPriv:         "Insert",
+	UpdatePriv:         "Update",
+	DeletePriv:         "Delete",
+	DropPriv:           "Drop",
+	GrantPriv:          "Grant",
+	ReferencesPriv:     "References",
+	LockTablesPriv:     "Lock Tables",
+	CreateTMPTablePriv: "Create Temporary Tables",
+	AlterPriv:          "Alter",
+	ExecutePriv:        "Execute",
+	IndexPriv:          "Index",
+	CreateViewPriv:     "Create View",
+	ShowViewPriv:       "Show View",
+	CreateRolePriv:     "Create Role",
+	DropRolePriv:       "Drop Role",
+	ShutdownPriv:       "Shutdown Role",
```

Comment:
(unrelated)

shouldn't this be

```suggestion
	ShutdownPriv:       "Shutdown",
```